src/app/components/cds/inputs/LanguagePickerSelect.js

```diff
@@ -70,8 +71,9 @@ const LanguagePickerSelect = ({
                 <TextField
                   iconLeft={<LanguageIcon />}
                   iconRight={<ChevronDownIcon />}
-                  label={showLabel ? <FormattedMessage id="LanguagePickerSelect.selectLanguageLabel" defaultMessage="Language" description="Label for input to select language of fact-check" /> : null}
+                  label={label && <FormattedMessage id="LanguagePickerSelect.selectLanguageLabel" defaultMessage="Language" description="Label for input to select language of fact-check" />}
```

Comment:
I guess this should be `label || <Formatt...` if you want `label` to display if set